hil/ext/switches/ovs.py

```diff
@@ -76,6 +77,10 @@ def disconnect(self):
     def validate_port_name(port):
         """ This driver accepts any string as port name."""
 
+    def get_capabilities(self):
+        """Provides the features of this switch that HIL supports. """
+        return['Virtual_switch_for_development_Purpose_only']
```

Comment:
this is not what we want to do with capabilities.

this method is supposed to return a list of things that the switch can support. Currently these return nothing, but one could return things like `per-port-jumbo-frames`, `qos`, etc. So in future an API implementing per port jumbo frames, or qos could just do 
```python
if capability not in switch_driver.get_capabilities():
    raise Exception("Not supported on this switch")
```
So just let it return `[]`